prime-router/docs/proposals/0009-proposal-for-history-api.md

```diff
@@ -0,0 +1,316 @@
+# DRAFT History API Proposal
+
```

Comment:
```suggestion

NOTE:   We have put this proposal on hold, while we explore options to provide data lineage, metadata, and similar user-facing information via a canned 3rd party query/viz tool, rather than via adding an API layer.  
```